src/github/client/github-client.ts

```diff
@@ -70,24 +69,37 @@ export default class GitHubClient {
 		return new AuthToken(tokenResponse.token, new Date(tokenResponse.expires_at));
 	}
 
-	/**
-	 * Lists pull requests for the given repository.
-	 */
-	public async getPullRequests(
-		owner: string,
-		repo: string,
-		pageSize: number,
-		page: number): Promise<Octokit.PullsListResponseItem[]> {
-		const response = await this.axios.get<Octokit.PullsListResponseItem[]>(`/repos/${owner}/${repo}/pulls`, {
+	private async get<T>(url, params = {}): Promise<AxiosResponse<T>> {
+		const response = await this.axios.get<T>(url, {
 			...await this.installationAuthenticationHeaders(),
-			params: {
+			params: { 
 				installationId: this.githubInstallationId,
-				per_page: pageSize,
-				page: page
+				...params
 			}
 		});
+		//TODO: error handling
```

Comment:
We can remove this comment from here hence error handling is not the responsibility of the HTTP Client